test/kustomize/stackdefinition.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: stacks.crossplane.io/v1alpha1
+kind: StackDefinition
+metadata:
+  name: template-stack-test
+spec:
+  behavior:
+    source:
+      image: crossplane/stack-minimal-gcp:0.0.2
+      path: "kustomize"
+    crd:
+      kind: MinimalGCP
+      apiVersion: gcp.resourcepacks.crossplane.io/v1alpha1
+    engine:
+      type: kustomize
+      kustomize:
```

Comment:
Wasn't able to leave a comment on the one in `test/helm3`, so I'm doing it here instead . . . do we want the `helm3` stackdefinition to use an engine type of `helm3`? Looks like it's using `kustomize` currently.

Reply:
Thanks for catching! Just updated them.